canonical-web-and-design/certification.ubuntu.com
# Adds images to homepage via the asset-server

# Done 
creates the homepage

Fixes https://github.com/canonical-web-and-design/base-squad/issues/621

# QA
`./run` and make sure that the homepage matches https://certification.ubuntu.com/
